TUISample/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Thu Jul 12 13:02:30 PDT 2018
-distributionBase=GRADLE_USER_HOME
-distributionPath=wrapper/dists
-zipStoreBase=GRADLE_USER_HOME
-zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
+#Tue Jan 15 14:05:47 PST 2019

```

Comment:
We have upgraded the other projects to gradle 4.10.2 now. This file has incorrect line endings, so I'm not sure if it was copied from somewhere else. Please upgrade again using this command: `./gradlew wrapper --gradle-version 4.10.2 --distribution-type all`
Then commit the files that the update changed.